lib/galaxy/webapps/galaxy/api/tool_shed_repositories.py

```diff
@@ -100,11 +100,10 @@ def get_latest_installable_revision( self, trans, payload, **kwd ):
         # Make sure the current user's API key proves he is an admin user in this Galaxy instance.
         if not trans.user_is_admin():
             raise exceptions.AdminRequiredException( 'You are not authorized to request the latest installable revision for a repository in this Galaxy instance.' )
-        params = '?name=%s&owner=%s' % ( name, owner )
-        url = common_util.url_join( tool_shed_url,
-                                    'api/repositories/get_ordered_installable_revisions%s' % params )
+        params = dict(name=name, owner=owner)
+        pathspec = 'api/repositories/get_ordered_installable_revisions'
```

Comment:
I agree it would be more consistent, but what was the reasoning behind that syntax? It seems counter-intuitive: the documentation above refers to the API path in string notation.